1907-04-22.xml

```diff
@@ -292,1250 +326,2389 @@
-                    <p>Arrive Alexandria Friday noon, leave Alexandria Tuesday, 3 p.m. Circular route
-                        between Alexandria, Port Said, the Syrian ports, Chio, Smyrna, Salonica, Mount
-                        Athos, Dardanelles, Constantinople, and Odessa. Arrive Alexandria alternate
-                        Saturdays early morn; leaving Alexandria same day at 4 p.m.</p>
-                    <p><hi rend="bold">Crimean or Bessarabian table wines free</hi>.</p>
-                    <p><measure type="indexNo">26376-31-6-907</measure></p>
-                </div><div type="advert" xml:id="deg-ad-ngi01" xml:lang="fr">
-                    <head>Navigation Générale Italienne.</head>
-                    <p>Societes Reunies Florio-Rubattino. - Services Postaux. - Departs d'Avril.</p>
-                    <table rows="5" cols="4">
-                        <row>
-                            <cell>Les Jeudis</cell>
-                            <cell>4, 11, 18, et 25</cell>
-                            <cell>à 3 h. p.m.</cell>
-                            <cell>direct pour Messine, Naples, Livourne et Gênes.</cell>
-                        </row>
-                        <row>
-                            <cell>Les Lundis</cell>
-                            <cell>1 et 15</cell>
-                            <cell>à 3 h. p.m.</cell>
-                            <cell>direct pour Brindisi, Bari, Ancône et Venise.</cell>
-                        </row>
-                        <row>
-                            <cell>Les Mercredis</cell>
-                            <cell>3 et 17</cell>
-                            <cell>à 10 h. a.m.</cell>
-                            <cell>pour les escales de la Syrie et Larnaque.</cell>
-                        </row>
-                        <row>
-                            <cell>Les Lundis</cell>
-                            <cell>1 et 29</cell>
-                            <cell>à 4 h. p.m.</cell>
-                            <cell>pour Port-Saïd, Suez et Massawah.</cell>
-                        </row>
-                        <row>
-                            <cell>Les Dimanche</cell>
-                            <cell>5 et Lundi 22</cell>
-                            <cell>à 5 h. p.m.</cell>
-                            <cell>pour Port-Saïd.</cell>
-                        </row>
-                    </table>
-                    <p><measure type="indexNo">31-12-906</measure></p>
-                </div><div type="advert" xml:id="deg-ad-wsl02">
-                    <head>WHITE STAR LINE.</head>
-                    <p>NEW YORK AND BOSTON MEDITERRANEAN SERVICE.</p>
-                    <p>REGULAR SAILINGS BETWEEN SOUTHAMPTON-CHEROUG and NEW YORK via Queenstown (Westbound)
-                        via Plymouth (Eastbound) LIVERPOOL and NEW YORK, via Queenstown, LIVERPOOL and BOSTON viq Queentown,
-                        GENOA, PALERMO,NAPLES, NEW YORK and
```

Comment:
Add a `</div>` here to close the local and general section. You'll have to remove a `</div>` from the end of the page.